addons/pos_mercury/views/pos_mercury_views.xml

```diff
@@ -78,6 +78,16 @@
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_pos_user_form" model="ir.ui.view">
```

Comment:
Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-a6c93aba12a2013af951703a88cbce10R71 .